clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -126,6 +126,11 @@ New checks
   reference. This may cause use-after-free errors if the caller uses xvalues as
   arguments.
 
+- New :doc:`bugprone-pointer-arithmetic-on-polymorphic-object
```

Comment:
add in line 173 info about new alias (check release notes from previous release)